src/dewey/curation.clj

```diff
@@ -348,6 +350,10 @@
               (irods-backoff)) ; This will retry forever
             (throw e)))
         (log/warn (str "unknown routing key" routing-key "received with message" msg)))
-      (let [consumetime (milliseconds-since consume-start)]
-        (tc/with-logging-context {:amqp-total-time consumetime :dewey-consume-time @innertime}
-          (log/info (format "[curation/consume-msg] [%s] [%s] %gms consume/%gms total" routing-key msg @innertime consumetime)))))))
+      (let [consumetime (milliseconds-since consume-start)
+            queue-stats (lq/status channel (cfg/queue-name))]
+        (tc/with-logging-context {:amqp-message-count (:message-count queue-stats)
+                                  :amqp-consumer-count (:consumer-count queue-stats)
```

Comment:
I'm not sure if these should include "queue" somewhere in their name, to be (potentially) clearer that they refer to one specific queue, or some other specifier.